### ISSUE TITLE
[Video player] Lack focus state blue outline around thumbnail before play

### ISSUE THREAD

**Issue by shixiedesign** [carbon-design-system/carbon-for-ibm-dotcom#5660]
### Problem
Video player's thumbnail should have a focus state so keyboard users can visually identify it's selected, and then they can expect pressing `spacebar` would trigger the video to play. See gif: we are missing the focus state on 2nd tab: 

![Mar-29-2021 20-30-24](https://user-images.githubusercontent.com/15144993/112916125-ecc59800-90cd-11eb-8553-08e8978fe94f.gif)

### Expected
Tab to focus on video player should trigger blue-60 focus border as below.

![image](https://user-images.githubusercontent.com/15144993/112916442-91e07080-90ce-11eb-8725-5983ceb3d9fa.png)

**Comment by RobertaJHahn** [comment]
@RaphaelAmadeu I have assigned this issue to you as it looks like a PR has been opened and Putra is working on the fix.

**Comment by shixiedesign** [comment]
Putra’s PR doesn’t fix this bug. It’s still present.

**Comment by DragosRistici** [comment]
A follow up after issue reopen:

1. Web Components Canary: focus state includes the caption text
<img width="607" alt="Screenshot 2021-04-12 at 12 32 19" src="https://user-images.githubusercontent.com/48327647/114373480-48425c00-9b8b-11eb-965c-c21fb9d1a14d.png">


2. React Canary: focus state not present
<img width="589" alt="Screenshot 2021-04-12 at 12 37 40" src="https://user-images.githubusercontent.com/48327647/114374039-e3d3cc80-9b8b-11eb-83a6-de8fd777bdd3.png">

3. Caption on gray 100 is 12px
<img width="176" alt="Screenshot 2021-04-12 at 12 34 37" src="https://user-images.githubusercontent.com/48327647/114373941-c868c180-9b8b-11eb-84cc-504f1b6533cb.png"> <img width="186" alt="Screenshot 2021-04-12 at 12 34 31" src="https://user-images.githubusercontent.com/48327647/114373946-c999ee80-9b8b-11eb-9bde-4290c5b4928e.png">

4. Was wondering if the Kaltura buttons focus color can be IDL'ed 
<img width="53" alt="Screenshot 2021-04-12 at 12 34 07" src="https://user-images.githubusercontent.com/48327647/114375189-18945380-9b8d-11eb-99ac-ce0394b97d85.png"> <img width="44" alt="Screenshot 2021-04-12 at 12 34 01" src="https://user-images.githubusercontent.com/48327647/114375193-192cea00-9b8d-11eb-95ad-0c15c272b5c2.png">
